avaframe/out3Plot/outAIMEC.py

```diff
@@ -1040,3 +1041,102 @@ def addLinePlot(contourDict, colorStr, labelStr, ax, key, zorder=''):
             zorder=zorder)
     else:
         ax.plot(contourDict['x'], contourDict['y'], c=colorStr, zorder=zorder)
+
+
+def plotThalwegAltitude(pathDict, rasterTransfo, pfvCrossMax, simName):
+    """ create thalweg-altitude plot
+        the thalweg z profile and the mpfv²/2g (velocity-altitude) colorcoded using the mpfv values
+        the mpfv values come from the cross max values along the thalweg coordinate system (aimec)
+        plot is saved to avaDir/Outputs/out3Plot/thalwegAltSimname
+
+        Parameters
+        ------------
+        avaDir: pathlib path
+            path to avalanche directory
+        rasterTransfo: dict
+            dict with info on transformation from cartesian to thalweg coordinate system
+        pfvCrossMax: pandas dataFrame series
+            cross profile max values of peak flow velocity transformed into thalweg coordinate system
+        simName: str
+            simulation name
+    """
+
+    # initialize figure
+    fig = plt.figure(figsize=(pU.figW, pU.figH))
+    ax1 = fig.add_subplot(1, 1, 1)
+
+    # add thalweg-altitude plot to axes
+    ax1 = addThalwegAltitude(ax1, rasterTransfo, pfvCrossMax)
+
+    # save and or plot
+    plotName = ('thalwegAlt%s' % (simName))
+    plotPath = pU.saveAndOrPlot(pathDict, plotName, fig)
+
+
+def addThalwegAltitude(ax1, rasterTransfo, pfvCrossMax, zMaxM=np.nan):
+    """ add thalweg-altitude plot to axes
+        the thalweg z profile and the mpfv²/2g (velocity-altitude) colorcoded using the mpfv values
+        the mpfv values come from the cross max values along the thalweg coordinate system (aimec)
+
+        Parameters
+        ------------
+        rasterTransfo: dict
+            dict with info on transformation from cartesian to thalweg coordinate system
+        pfvCrossMax: pandas dataFrame series
+            cross profile max values of peak flow velocity transformed into thalweg coordinate system
+        zMaxM: float
+            optional - value to define max limit of y -axis
+    """
+
+    # compute velocity-Altitude-Field
+    g = 9.81  # gravitation constant
```

Comment:
Mh, I don't like this here. We also have it in com1DFA. I see the following options:
1. use the com1DFA one
2. have it in the avaframeCfg.ini available for all
3. add it it the outAIMEC cfg

I'm not sure whether I prefer 2. or 3.